src/queries/index.js

```diff
@@ -126,16 +126,39 @@ export const buildPresenceQuery = sourceAccountIdentifier => ({
   }
 })
 
-export const buildAccountFolderQuery = accountIdentifier => ({
+export const buildAccountQuery = () => ({
   definition: () =>
-    Q('io.cozy.files')
-      .partialIndex({ type: 'directory', trashed: false })
-      .referencedBy({
-        _id: accountIdentifier,
-        _type: 'io.cozy.accounts.sourceAccountIdentifier'
+    Q('io.cozy.accounts')
+      .where({
+        account_type: 'pronote'
+      })
+      .partialIndex({
+        account_type: 'pronote'
+      }),
+  options: {
+    as: 'io.cozy.files/account_type/pronote',
+    fetchPolicy: defaultFetchPolicy,
+    singleDocData: true
+  }
+})
+
+export const buildTriggerQuery = accountId => ({
+  definition: () =>
+    Q('io.cozy.triggers')
+      .where({
+        message: {
+          account: accountId,
+          konnector: 'pronote'
+        }
+      })
+      .partialIndex({
+        message: {
+          account: accountId,
+          konnector: 'pronote'
+        }
       }),
   options: {
-    as: 'io.cozy.files/byReferencedAccountIdentifier/' + accountIdentifier,
+    as: 'io.cozy.triggers/konnector/pronote/account/' + accountId,
     fetchPolicy: defaultFetchPolicy,
     singleDocData: true
```

Comment:
There is a `singleDocData` here but inside the hook you select the first item into a row. Maybe we should remove this option to avoid misleading the futur developer. The same note is valuable for `buildAccountQuery`

Reply:
Updated 👍